src/Components/Components/ref/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -3,6 +3,34 @@
 
 namespace Microsoft.AspNetCore.Components
 {
+    public partial class AuthenticationState
+    {
+        public AuthenticationState(System.Security.Claims.ClaimsPrincipal user) { }
+        public System.Security.Claims.ClaimsPrincipal User { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
+    }
+    public delegate void AuthenticationStateChangedHandler(System.Threading.Tasks.Task<Microsoft.AspNetCore.Components.AuthenticationState> task);
+    public abstract partial class AuthenticationStateProvider
+    {
+        protected AuthenticationStateProvider() { }
+        public event Microsoft.AspNetCore.Components.AuthenticationStateChangedHandler AuthenticationStateChanged { add { } remove { } }
+        public abstract System.Threading.Tasks.Task<Microsoft.AspNetCore.Components.AuthenticationState> GetAuthenticationStateAsync();
+        protected void NotifyAuthenticationStateChanged(System.Threading.Tasks.Task<Microsoft.AspNetCore.Components.AuthenticationState> task) { }
+    }
+    public partial class AuthorizeView : Microsoft.AspNetCore.Components.ComponentBase
```

Comment:
There's a high level question that came up for me. I'm going to mention it, and I see it as more of an API review thing rather than something to address right now.

We should develop the vocabulary we want to use for components that we provide. Like this guy is the `AuthorizeView`, it could also be the `AuthorizeDisplay` or something else. 

I indexed on the work `View` because that already means something in MVC. Anyways, just for discussion 👍

Reply:
I think that’s a hood idea, although I don’t necessarily subscribe your specific suggestion.

Reply:
Likewise I agree. TBH I think the name `AuthorizeDisplay` would be fine. However we can leave this as-is until the API review.